setup-claude-server.js

```
@@ -141,615 +163,659 @@ async function enhancedGetTrackingProperties(additionalProps = {}) {
+      resolve(false);
+    }, timeoutMs);
 
+    try {
+      await trackEvent(eventName, additionalProps);
+      clearTimeout(timeoutId);
+      resolve(true);
+    } catch (error) {
+      clearTimeout(timeoutId);
+      resolve(false);
+    }
+  });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)